### PR TITLE
Introduce pandas

### PR DESCRIPTION
**Pull request by Kobo-coder**
## Describe your changes

I have not tested the code as this is a change to visualization. I have run the program and seen that I get the expected graphs with the data given in csv files. If you run main on this branch and it shows empty graphs with weird units on the x-axis instead of dates, it is most likely due to differences in the csv file format.

This PR fixes both the date time on bokeh plots and introduces pandas as a way to read csv files.

## Checklist before requesting a review

- [x] I have performed a self-review of my code.
- [ ] If it is a core feature, I have added thorough tests.
- [x] Have I requested and notified the boys for review of the pull-request?

### CODE REVIEW

**Review by SadCheeto — APPROVED** [review]

File: byoqm/visuals/dashboard.py [hunk 1]
```diff
@@ -10,31 +12,17 @@ class Dashboard:
     def show_graphs(self):
         data = self.get_data()
         f = [line.get_line(data, key) for key in data]
-        grid = gridplot([[f[0], f[1]], [f[2], f[3]], [f[4], f[5]]])
+        grid = gridplot([[f[i], f[i + 1]] for i in range(0, len(f) - 1, 2)])
```
Comment (toffernator) [inline byoqm/visuals/dashboard.py:15]: Some more descriptive names here would be nice, because I have no clue what is going on.

Comment (Kobo-coder) [reply byoqm/visuals/dashboard.py:15]: ok

**Review by FrederikRothe — APPROVED** [review]
Good job 👍🏼

File: byoqm/visuals/dashboard.py (hunk 1 above)
Comment (SadCheeto) [reply byoqm/visuals/dashboard.py:15]: will changing f -> figures be enough? Else I would argue that a bit of documentation is all that is needed

**Pull request merged**